src/components/common/useRequest.ts

```diff
@@ -53,3 +55,7 @@ export function useRequest<T = unknown>(
     refreshWhenOffline: true,
   });
 }
+
+function isCloudWorkstation(url: string) {
```

Comment:
this is also defined in rest_api, did you mean to have two implementations?

Reply:
Nope, this one was actually unused - i should be more careful with self review lol